From 24ebd3328c611e187874b5a446c30d88b687aab5 Mon Sep 17 00:00:00 2001
From: Nic Costa <nic.costa@arm.com>
Date: Wed, 25 Oct 2017 09:24:00 -0500
Subject: [PATCH] Increase bootloader partition size on EVK-ODIN-W2

With our FOTA patches, the bootloader is too large to fit
within the 131072 size boundary.  Increase the size to the
next available write block size.

Signed-off-by: Nic Costa <nic.costa@arm.com>
---
 mbed_app.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mbed_app.json b/mbed_app.json
index d64056e..f70e306 100755
--- a/mbed_app.json
+++ b/mbed_app.json
@@ -45,8 +45,8 @@
         },
         "UBLOX_EVK_ODIN_W2": {
             "target.device_has_remove": ["EMAC"],
-            "firmware_metadata_header_address": "0x08020000",
-            "update-client.application-details": "0x08020000",
+            "firmware_metadata_header_address": "0x08040000",
+            "update-client.application-details": "0x08040000",
             "firmware_metadata_header_size": "0x400"
         }
     }
-- 
2.14.3

